### PR TITLE
Refactor Image Toolbar to use Grommet Button and Blank icon

### PR DESCRIPTION
**Pull request by mcbouslog**
Package: lib-classifier

Describe your changes:
- refactors image toolbar icons with Grommet `<Blank />` icon component
- refactors Button.js with styled Grommet `<Button />`
- adds `title` attribute for temporary tooltip until Grommet `<Drop />` fixed (see Grommet issue 2660)

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

**Comment by srallen** [comment]
@mcbouslog and I will pair on resolving the Grommet bug this week, but we won't know the timeline on that being reviewed and published, so using HTML title attr is a good interim.

**Comment by srallen** [comment]
@mcbouslog There's something wrong with the travis build: https://travis-ci.com/zooniverse/front-end-monorepo/builds/110393301

It looks like some components aren't being found?

**Comment by mcbouslog** [comment]
TIL renaming a file with only a capitalization change may or may not be recognized by git depending on OS, in which case `git mv --force myFile MyFile` forces the update.

File: packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/Button/Button.js [hunk 1]
```diff
@@ -1,65 +1,43 @@
...
-      width: `${sizeInPercent}%`
+    > svg {
+      fill: white;
     }
   }
 
+  > svg {
+    fill: ${props => props.active ? 'white' : (props.theme.dark ? 'white' : 'black')};
+    stroke: transparent;
+    width: ${pxToRem(18)};
```
Comment (srallen) [inline packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/Button/Button.js]: Height needs to be set as well here. Right now, the icons are inheriting the Grommet default for height now, 24px, and it's making the circle buttons oval now. Height should be the same as width to have the inner icon be square and the outer button be a circle.

Comment (mcbouslog) [reply packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/Button/Button.js]: Yes it looked a little off thank you for catching!

**Pull request merged**